src/feature_transform_executor.rs

```diff
@@ -268,7 +265,7 @@ mod tests {
             namespace_seeds: default_seeds(1), // These are precomputed namespace seeds
             tmp_data: Vec::new(),
         };
-        let mut to_namespace = to_namespace_empty.clone();
```

Comment:
@bbenshalom this clone was removed as the object is actually cloned as part of init of the mutable version + it's scope ends as part of this test anyways (few updates are the only thing happening to it)